constraint/pkg/apis/addtoscheme_templates_v1alpha1.go

```diff
@@ -15,11 +15,7 @@ limitations under the License.
 
 package apis
 
-import (
-	"github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1alpha1"
-)
-
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1alpha1.AddToSchemes...)
```

Comment:
I'm a bit confused here. Why are we appending a list to itself?

Should this not read as:

`AddToSchemes = append(AddToSchemes, v1alpha1.AddToScheme)`

?

Reply:
@maxsmythe thanks, it should be fixed now